test.js

```diff
@@ -2,8 +2,10 @@
  * Module dependencies.
  */
 
-var assert = require('assert');
-var validate = require('./');
+var assert = require('assert')
+var validate = require('./')
+
+assert.deepStrictEqual = require('is-equal-shallow')
```

Comment:
Ah, I understand your reasoning now. Still I prefer not to extend an existing library since we've not ditched `0.10` yet and it can confuse people looking at the code (like it did with me).